backend/routes/api/file.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const router = express.Router();
+
+const multer = require('multer');
+const storage = multer.diskStorage({
+    destination: 'files/',
+    filename: function(req, file, callback) {
+        let ext = file.originalname.split('.').slice(-1);
+        callback( null, Date.now() + '.' + ext ); // 1633269573153.ext
```

Comment:
How about appending the original file name or a random hash to the end of the `fileName` in order to avoid duplicate names? The chances are extremely low, but just to be safe. 😉

Reply:
Resolved